### PR TITLE
Remove custom org.openbmc.Object.ObjectMapper usage

### PR DESCRIPTION
**Pull request by bradbishop**
This change is [Reviewable](https://reviewable.io/reviews/openbmc/skeleton/61)

### CODE REVIEW

File: includes/openbmc.h [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #ifdef __arm__
 static inline void devmem(void* addr, uint32_t val)
 {
-	printf("devmem 0x%08x = 0x%08x\n",addr,val);
+	printf("devmem 0x%08x = 0x%08x\n",(uint32_t)addr,val);
```
Comment (williamspatrick) [inline includes/openbmc.h]: Better to change the first %08x to a %08p rather than casting.

**Comment by williamspatrick** [comment]
+1

File: includes/openbmc.h (hunk 1 above)
Comment (bradbishop) [inline includes/openbmc.h]: fixed.

**Comment by bradbishop** [comment]
Cyril had some comments on the list too...which I also incorporated with the latest sync.

**Pull request merged**